primitives/src/vid.rs

```diff
@@ -84,6 +84,23 @@ pub struct VidDisperse<V: VidScheme + ?Sized> {
     pub commit: V::Commit,
 }
 
+/// [`VidScheme::Common`] could impl this trait to allow users to get the
```

Comment:
Can we unify the two traits into a single trait called `CommonData`?

Reply:
Discussed offline. Fixed in https://github.com/EspressoSystems/jellyfish/pull/450/commits/564765e6cf87fe03ec52f4af90b1cc642e3d415a